File: 프로그래머스/1/12901. 2016년/2016년.js
function solution(a, b) {
    var answer = '';
    const obj = {
        0 : "SUN",
        1 : "MON",
        2 : "TUE",
        3 : "WED",
        4 : "THU",
        5 : "FRI",
        6 : "SAT",
    }
    let date = new Date(`2016-${a}-${b}`);
    answer = obj[date.getDay()];
    return answer;
}

File: 프로그래머스/1/12901. 2016년/2016년.py
from datetime import datetime

def solution(a, b):
    
    dic = {0:"MON" , 1:"TUE" , 2:"WED" , 3:"THU" , 4:"FRI" , 5:"SAT" , 6:"SUN"}
    date = datetime(2016,a,b)
    
    return dic[date.weekday()]